ci/edmtool.py

```diff
@@ -104,25 +103,24 @@
         "pyparsing",
         "pypdf2",
         "reportlab",
-        "swig",
         "traits",
         "traitsui",
+        "wheel",
```

Comment:
This is a somewhat independent change: the install of pygarrayimage from PyPI was complaining about having to use a legacy install because `wheel` wasn't present.